fasteners/process_lock.py

```diff
@@ -214,30 +214,44 @@ def exists(self):
         return os.path.exists(self.path)
 
     def trylock(self):
-        raise NotImplementedError()
+        self._trylock(self.lockfile)
 
     def unlock(self):
+        self._unlock(self.lockfile)
+
+    @staticmethod
```

Comment:
Curious, any reason to add these vs just using the existing trylock and unlock methods?